test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -1856,7 +1858,10 @@ TEST_F(ClusterManagerImplTest, MergedUpdatesOutOfWindow) {
   HostVector hosts_removed;
 
   // The first update should be applied immediately, because even though it's mergeable
-  // it's outside a merge window.
```

Comment:
I was a little curious how this was working before, comparing real-time against a 3 second delay, but after debugging this in a clean client I realized that no prior update had ever occurred, so the duration was measured against a start-time of 0. In other words, this test passed if the machine was rebooted more than 3 seconds prior to running this test.